hello-key/config.ml

```diff
@@ -0,0 +1,14 @@
+open Mirage
+
+let key =
+  let doc = Key.Arg.info ~doc:"How to say hello." ["hello"] in
+  Key.(create "hello" Arg.(opt string "Hello World!" doc))
+
+let main =
+  foreign
+    ~keys:[Key.abstract key]
```

Comment:
is it too late to rename `Key.abstract` into `Key.v`? :-) /cc @yomimono and @Drup